lessons/14-css-more-rules/index.html

```diff
@@ -0,0 +1,55 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>CSS Rules</title>
+
+    <style>
+        .first-div{background: red;
+            padding: 20px;
+            display: inline-block;
+            border: 1px solid orange;}
+
+        .second-div{background:blue;
+            padding-left: 10px;
+            padding-right: 10px;
+            padding-top: 30px;
+            padding-bottom: 30px;
+
+            display: inline-block;
+            border: 4px solid green;}
+
+        .third-div{background: #00aced;
+            height: 50px;
+            width: 50px;}
+
+        .forth-div{
+            display: none;
+            }
```

Comment:
Looking good! One recommendation is just to keep the properties neat and defined on separate lines

Before:

```css
.second-div{background:blue;
            padding-left: 10px;
            padding-right: 10px;
            padding-top: 30px;
            padding-bottom: 30px;
            display: inline-block;
            border: 4px solid green;}
```

After

```css
.second-div {
  background:blue;
  padding-left: 10px;
  padding-right: 10px;
  padding-top: 30px;
  padding-bottom: 30px;
  display: inline-block;
  border: 4px solid green;
}
```